content-src/components/DiscoveryStreamComponents/SectionTitle/_SectionTitle.scss

```diff
@@ -1,30 +1,15 @@
 .ds-section-title {
-  ul {
-    margin: 0;
-    padding: 0;
-  }
-
-  ul li {
-    display: inline-block;
-
-    &::after {
-      content: '•';
-      padding: 8px;
-    }
+  text-align: center;
 
-    &:last-child::after {
-      content: none;
-    }
+  .title {
+    line-height: 48px;
+    font-size: 36px;
```

Comment:
Do these need different sizes for other column sizes?

Reply:
Technically the comps only have this defined with 1 width. I don't think it's their intent to use the title in the 1/3 and 2/3 views, but if they decide to, yeah, adding a size update might be worth it, otherwise, it'll still do something not terrible.